metrics/storage/fio_job.sh

```diff
@@ -161,14 +194,42 @@ function main()
 
 	init_env
 	check_cmds "${cmds[@]}"
-	create_fio_job
+	create_fio_run_job
+	create_fio_prep_job
 
-	# Launch container
-	output=$(docker run --runtime="$RUNTIME" \
-		"$FIO_IMAGE" bash -c "$FIO_JOB")
+	# First we set up the container
+	CONTAINER_ID=$(docker run -tid --runtime="$RUNTIME" "$FIO_IMAGE" bash)
+
+	# Now we run the prep task
+	docker exec -ti ${CONTAINER_ID} bash -c "$FIO_PREP_JOB"
+
+	# Kick off background tasks to measure the CPU, PSS and RSS
+	cpu_temp=$(mktemp fio_job_cpu.XXX)
+	pss_temp=$(mktemp fio_job_pss.XXX)
+	rss_temp=$(mktemp fio_job_rss.XXX)
+
+	# We know the test will run for 30s (unless somebody changed it...)
+	# So spread the measures out over the test run
+	(sleep 10; ps --no-headers -o %cpu -C $(basename ${QEMU_PATH}) > ${cpu_temp})&
```

Comment:
I made a test and it shows and error using `runc`, it seems that the test is looking for a `qemu` instance, maybe we could check the runtime and in the case of `runc` we can measure the `fio` process running in host

Reply:
Yeah. I knew this was a gap when I did the PR - I was going to ask @GabyCT what 'process' she measured when not running a VM runtime - and I guess the logical answer is the `fio` process - so, sure, let me make that modification...

Reply:
@GabyCT can you confirm which process is used in the case of runc?